src/ngraph/op/group_convolution.cpp

```diff
@@ -0,0 +1,331 @@
+//*****************************************************************************
```

Comment:
Why is this not a fused op?

Reply:
The original fused op was already used in passes, kernels and template functions.
Since name of the attributes(and his getters) would change it will make conflicts.
That's why I made a decision to make a new op instead of adjusting the existing fused operator.
The work on these operators started when fused ops didn't use namespaces for opsets.
We can consider to make these operators a v1 fused ops again but I would prefer to do it in another PR.